### ISSUE TITLE
Soybean, Wisteria, Seaweed and Lotus Root grow abnormally

### ISSUE THREAD

**Issue by RisingInIris2017** [defeatedcrow/HeatAndClimateMod#121]
There are two pictures for this issue:
![](https://i.imgur.com/GcRBxnM.jpg)
Soybean crops can grow on another soybean crops, simply by right click on a soybean crop with a soybean seed in hand.
Wisteria has the same issue.

![](https://i.imgur.com/f6Zc2w3.jpg)
Lotus and seaweeds crops look a bit weird.

**Comment by defeatedcrow** [comment]
They are all according to the specification. It has the behavior I intended.

About Seedlings stacked: It can be stacked by players, just like vanilla reeds. I did it.
About Lotus: In order to make the drawing strict, the condition of the adjacent block becomes hard. That is an unnecessary increase in difficulty.
Or it needs a modification of Block Render, but it is difficult to do that with json. (Modification would have been easy if Block Render prior to 1.7 remained, but it has already been lost.)
